packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -34,6 +34,7 @@ const ALLOWED_BLOCKS = [
 	'core/site-title',
 	'core/site-logo',
 	'core/navigation-submenu',
+	'core/loginout',
```

Comment:
We should normalize these lists to a single `const` and import into the relevant files.

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/50323